clang/include/clang/Driver/Options.td

```diff
@@ -1306,6 +1306,14 @@ defm cuda_prec_sqrt : BoolFOption<"cuda-prec-sqrt",
 def emit_static_lib : Flag<["--"], "emit-static-lib">,
   HelpText<"Enable linker job to emit a static library.">;
 
+def fpreview_major_release : Flag<["-"], "fpreview-breaking-changes">, Flags<[NoXarchOption]>,
```

Comment:
variable def should match the option name for consistency
```suggestion
def fpreview_breaking_changes : Flag<["-"], "fpreview-breaking-changes">, Flags<[NoXarchOption]>,
```

Reply:
My bad. It has been renamed properly.